forms-flow-api/src/formsflow_api/models/tasks.py

```diff
@@ -0,0 +1,41 @@
+"""This manages Task Data."""
+
+from __future__ import annotations
+
+from sqlalchemy import JSON
+
+from .audit_mixin import ApplicationAuditDateTimeMixin
+from .base_model import BaseModel
+from .db import db
+
+
+class TaskOutcomeConfiguration(ApplicationAuditDateTimeMixin, BaseModel, db.Model):
+    """This class manages task outcome configurations."""
+
+    id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(
+        db.String(100), nullable=False, comment="Task ID", unique=True, index=True
+    )
+    task_name = db.Column(db.String(100), nullable=True, comment="Task name")
+    task_transition_map = db.Column(
+        JSON, nullable=False, comment="Task transition configuration"
+    )
+    transition_map_type = db.Column(
+        db.String(100),
+        nullable=False,
+        default="select",
+        comment="Task transition configuration type",
+    )
+    created_by = db.Column(db.String(100), nullable=False, comment="Created by")
+    tenant = db.Column(db.String(100), nullable=True, comment="Tenant key", index=True)
```

Comment:
instead of adding index in tenant can we create composite indexing
__table_args__ = (
    db.Index("ix_tenant_task_id", "tenant", "task_id"),
)
because we are checking taskId and tenantId right if the tenant is not null

Reply:
done